doc/index.rst

```diff
@@ -13,6 +13,7 @@ Welcome to the Cabal User Guide
    :numbered:
    :maxdepth: 2
 
+   how-to-source-packages
```

Comment:
Please move the new guid to the second position as the usual dependency handling is much more frequent.

Reply:
:+1:

Reply:
Maybe the commit got lost; it's still showing 
`how-to-source-packages` before `how-to-package-haskell-code`